Fitness/Controllers/HomeController.swift

```diff
@@ -160,6 +167,10 @@ extension HomeController: UICollectionViewDataSource {
 
             }
             return cell
+        } else if collectionView.accessibilityIdentifier == Identifiers.discoverProsCell {
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: ProCell.identifier, for: indexPath) as! ProCell
+            cell.setPro(pro: pros[indexPath.row])
```

Comment:
`setPro(pro:)` could be renamed to `set(pro:)` or `setPro(_ pro:)`

Reply:
Understood, will change for the next commit